create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -454,38 +475,58 @@ private ArrayNode createPublicationChannelsResponseWithJournalUri(URI journalUri
     }
 
     private AdditionalIdentifier[] convertScopusIdentifiersToNvaAdditionalIdentifiers(
-            List<ItemidTp> scopusIdentifiers) {
+        List<ItemidTp> scopusIdentifiers) {
         return scopusIdentifiers
-                .stream()
-                .map(idtp -> new AdditionalIdentifier(ADDITIONAL_IDENTIFIERS_SCOPUS_ID_SOURCE_NAME, idtp.getValue()))
-                .collect(Collectors.toList())
-                .toArray(AdditionalIdentifier[]::new);
+            .stream()
+            .map(idtp -> new AdditionalIdentifier(ADDITIONAL_IDENTIFIERS_SCOPUS_ID_SOURCE_NAME, idtp.getValue()))
+            .collect(Collectors.toList())
+            .toArray(AdditionalIdentifier[]::new);
     }
 
     private List<ItemidTp> keepOnlyTheScopusIdentifiers() {
         return scopusData.getDocument()
-                .getItem()
-                .getItem()
-                .getBibrecord()
-                .getItemInfo()
-                .getItemidlist()
-                .getItemid()
-                .stream()
-                .filter(identifier -> identifier.getIdtype()
-                        .equalsIgnoreCase(ScopusConstants.SCOPUS_ITEM_IDENTIFIER_SCP_FIELD_NAME))
-                .collect(Collectors.toList());
+            .getItem()
+            .getItem()
+            .getBibrecord()
+            .getItemInfo()
+            .getItemidlist()
+            .getItemid()
+            .stream()
+            .filter(identifier -> identifier.getIdtype()
+                .equalsIgnoreCase(ScopusConstants.SCOPUS_ITEM_IDENTIFIER_SCP_FIELD_NAME))
+            .collect(Collectors.toList());
+    }
+
+    private Optional<Contributor> findContributor(String orcid, List<Contributor> contributors) {
+        return contributors.stream()
+            .filter(contributor -> orcid.equals(contributor.getIdentity().getOrcId()))
+            .findFirst();
+    }
+
+    private List<AuthorTp> keepOnlyTheAuthors() {
+        return keepOnlyTheCollaborationsAndAuthors().stream().filter(
+            this::isAuthorTp).map(author -> (AuthorTp) author).collect(Collectors.toList());
+    }
+
+    private boolean isAuthorTp(Object object) {
+        return object instanceof AuthorTp;
+    }
+
+    private List<Object> keepOnlyTheCollaborationsAndAuthors() {
+        return scopusData.getDocument().getItem().getItem().getBibrecord().getHead().getAuthorGroup().stream().map(
```

Comment:
add line breaks after each method call for readablity

Reply:
fixed